tests/create_buckets.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+from boto3 import Session
+from kubernetes import client, config
+import time
+import sys
+import os
+
+
+def get_env(key, default=None, error=False):
+    if not error:
+        return os.environ.get(key, default)
+    return os.environ[key]
+
+
+def bucket_safe_create(bucket):
+    try:
+        bucket.create()
+    except bucket.meta.client.exceptions.BucketAlreadyOwnedByYou:
+        print('Bucket %s already exists!' % bucket.name)
+    except Exception as exp:  # pylint: disable=broad-except
+        print('Error creating bucket %s - %s' % (bucket.name, str(exp)))
+        raise exp
+
+
+TIMEOUT = 120  # 2 Minutes
+
+K8S_NAMESPACE = os.getenv('ZENKO_K8S_NAMESPACE')
+ZENKO_HELM_RELEASE = os.getenv('ZENKO_HELM_RELEASE')
+ZENKO_ENDPOINT = get_env('CLOUDSERVER_FRONT_ENDPOINT',
+                         'http://%s-cloudserver-front:80' % ZENKO_HELM_RELEASE)
+VERIFY_CERTIFICATES = get_env('VERIFY_CERTIFICATES', False)
```

Comment:
Why do we need to set this to false and especially since s3 client is talking to zenko over http?

Reply:
For configuration Right now we're using http, however that could change in the future. Mostly the code for loading config was copy/pasted from the e2e tests, kinda a "if it ain't broke, don't fix it' situation

Reply:
I mean ideally we should not have unused code